Common/Currencies.cs

```diff
@@ -145,11 +145,11 @@ public static class Currencies
             {"LSK", "Ⱡ"},
             {"NAV", "Ꞥ"}
         };
-
+        
         /// <summary>
-        /// Stable pairs in GDAX. We defined them because they have different fees in GDAX market
+        /// Stable pairs in Coinbase. We defined them because they have different fees in Coinbase market
         /// </summary>
-        public static HashSet<string> StablePairsGDAX = new HashSet<string>
+        public static HashSet<string> StablePairsCoinbase = new()
```

Comment:
Shoulld be keep `StablePairsGDAX` and deprecate it in case users are accessing this field?

Reply:
Interesting question 🤔
I thought, It has used only for fees
![image](https://github.com/QuantConnect/Lean/assets/45608740/7e2aba33-71b4-437e-9f74-27844a632b78)

Reply:
Sure, but it's public, so perhaps users might be using it in some of their algorithms? Just making sure we don't break anything for anyone